src/mmw/js/src/draw/views.js

```diff
@@ -636,19 +636,31 @@ var DrawAreaView = DrawToolBaseView.extend({
         return {
             id: this.id,
             title: 'Draw area',
-            info: 'Lorem ipsum dolor it amet, consectetur',
+            info: 'Free draw an area or place a square kilometer',
             items: [
                 {
                     id: freeDraw,
                     title: 'Free draw',
-                    info: 'Draw a custom area on which you want to perform water quality analysis.',
+                    info: 'Free draw an area of interest polygon, by clicking ' +
+                          'on map and repeatedly clicking at boundary corners. ' +
+                          'Close the polygon by double clicking on the last ' +
+                          'point or clicking on the first point.<br />' +
+                          'For more information, see ' +
+                          '<a href=\'https://wikiwatershed.org/documentation/mmw-tech/#draw-area\' target=\'_blank\' rel=\'noreferrer noopener\'>' +
+                          'Model My Watershed Technical Documentation on ' +
+                          'Draw Area.</a>',
                     minZoom: 0,
                     directions: 'Draw a boundary.'
                 },
                 {
                     id: squareKm,
                     title: 'Square Km',
-                    info: 'Draw a predefined one kilometer square with a center at the point you choose.',
+                    info: 'Draw a perfect square with one kilometer sides, by ' +
+                          'clicking on map at the square’s center.<br />' +
```

Comment:
I also find this part of the description a little confusing:  `by clicking on map at the square’s center`. It's referencing the location of the yet undrawn square as though it's already on the map. The action is really `clicking on the map where the square's center should be`, though that doesn't sound any better.